ios/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array/>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>3B52.1</string>
+				<string>0A2A.1</string>
```

Comment:
0A2A.1: "Declare this reason if your third-party SDK is providing a
      wrapper function around file timestamp API(s) for the app to use,
      and you only access the file timestamp APIs when the app calls
      your wrapper function. This reason may only be declared by
      third-party SDKs. This reason may not be declared if your
      third-party SDK was created primarily to wrap required reason
      API(s).
    
      Information accessed for this reason, or any derived information,
      may not be used for your third-party SDK’s own purposes or sent
      off-device by your third-party SDK."
    
      expo-file-system (using the privacy manifest added recently in
        expo/expo@b8e40614b, which we haven't taken because Expo
        upgrades take a lot of effort)

I think this reason is appropriate for `expo-file-system`'s privacy manifest, but not for ours. As that doc says:
> This reason may only be declared by third-party SDKs.

Instead, I think the appropriate thing for our app is to declare whatever reasons apply to our use of that wrapper function. I believe we don't ultimately use that wrapper function at all, so that means we just leave this reason out and don't add any others in its place.

(The other two reasons still belong here, because it's possible we do use these APIs in those ways.)

Reply:
> I believe we don't ultimately use that wrapper function at all,

Concretely:

The one place `expo-file-system` appears to expose a file timestamp is in its `FileInfo` type, returned only by its `getInfoAsync`:
```
$ rg -iC12 time node_modules/expo-file-system/src/
$ rg -C4 FileInfo node_modules/expo-file-system/src/
```

The only references to that function in our code or dependencies are in `expo-file-system` itself, in `jest-expo` which is dev-only, and in `expo-asset`:
```
$ rg -wl getInfoAsync src/ node_modules/
```

The mentions within `expo-file-system` are all part of the implementation of the function, not call sites. And the one call site in `expo-asset` doesn't look at the timestamp:
```
$ rg -wC2 getInfoAsync node_modules/expo-asset/src/
```

So even if we do ever invoke that code in `expo-asset` (which I suspect we don't) and so call that wrapper function, we never look at the resulting timestamp.

Reply:
Cool, makes sense; I'll note this briefly in the commit message and link here for the explanation. Thanks!